docs/kh2/file/type/preferences.md

```diff
@@ -0,0 +1,284 @@
+| 80 | float | Lock On Release Range
+| 84 | float | Stun Recov
+| 88 | float | Stun Recov Hp
+| 92 | float | Stun Relax
+| 96 | float | Drive Zako
+| 100 | float | Change Time Zako
+| 104 | float | Drive Time
+| 108 | float | Drive Time Relax
+| 112 | float | Change Time Add Rate
+| 116 | float | Change Time Sub Rate
+| 120 | float | Mp Drive Rate
+| 124 | float | Mp To Mp Drive
+| 128 | float | Summon Time Relax
+| 132 | float | Summon Pray Time
+| 136 | float | Summon Pray Time Skip
+| 140 | int | Anti Form Drive Count
+| 144 | int | Anti Form Sub Count
+| 148 | float | AntiForm Damage Rate
+| 152 | float | Final Form Rate
+| 156 | float | Final Form Mul Rate
+| 160 | float | Final Form Max Rate
+| 164 | int | Final Form Sub Count
+| 168 | float | Attack Distance To Speed
+| 172 | float | Al Carpet Dash Inner
+| 176 | float | Al Carpet Dash Delay
+| 180 | float | Al Carpet Dash Accel
+| 184 | float | Al Carpet Dash Brake
+| 188 | float | Lk Dash Drift Inner
+| 192 | float | Lk Dash Drift Time
+| 196 | float | Lk Dash Accel Drift
+| 200 | float | Lk Dash Accel Stop
+| 204 | float | Lk Dash Drift Speed
+| 208 | float | Lk Magic Jump
+| 212 | float | Mickey Charge Wait
+| 216 | float | Mickey Down Rate
+| 220 | float | Mickey Min Rate
+| 224 | float | Lm Swim Speed
+| 228 | float | Lm Swim Control
+| 232 | float | Lm Swim Accel
+| 236 | float | Lm Dolphin Accel
+| 240 | float | Lm DolphinSpeedMax
+| 244 | float | Lm DolphinSpeedMin
+| 248 | float | Lm Dolphin Speed Max Distance
+| 252 | float | Lm Dolphin Speed Min Distance
+| 256 | float | Lm Dolphin Rotation Max
+| 260 | float | Lm Dolphin Rotation Distance
+| 264 | float | Lm Dolphin Rotation Max Distance
+| 268 | float | Lm Dolphin Distance To Time
+| 272 | float | Lm Dolphin Time Max
+| 276 | float | Lm Dolphin Time Min
+| 280 | float | Lm Dolphin Near Speed
+| 284 | int | Drive Berserk Attack
+| 288 | float | Mp Haste
+| 292 | float | Mp Hastera
+| 296 | float | Mp Hastega
+| 300 | float | Draw Range
+| 304 | int | Combo Damage Up
+| 308 | int | Reaction Damage Up
+| 312 | float | Damage Drive
+| 316 | float | Drive Boost
+| 320 | float | Form Boost
+| 324 | float | Exp Chance
+| 328 | int | Defender
+| 332 | int | Element Up
+| 336 | float | Damage Aspir
+| 340 | float | Hyper Heal
+| 344 | float | Combination Boost
+| 348 | float | Prize Up
+| 352 | float | Luck Up
+| 356 | int | Item Up
+| 360 | float | Auto Heal
+| 364 | float | Summon Boost
+| 368 | float | Drive Convert
+| 372 | float | Defense Master
+| 376 | int | Defense Master Ratio
+
+### magi
+Each pointer leads to a specific entry's offset.
+
+### magi Structure
+
+| Amount | Description |
+|--------|---------------|
+| 1 	 | Pointer count [uint]
+| 36 	 | Pointers [uint]
+| 5 	 | magi entries
+
+### magi Entry
+
+\* NOTE: Each entry is 124 bytes long, so the info for the fields is wrong somewhere.
+
+| Offset | Variable Type | Description |
+|--------|---------------|-------------|
+| 0 | float | Fire radius
```

Comment:
I am being picky here, but can you make these entries formatted in PascalCase? If course, with spaces. It seems to be inconsistent with other documentations this way.

Reply:
No, You are Not Being Picky Dear. It Is The Right Choice.

Reply:
Nice Use Of Pascal Case. I Approb!